packages/react/src/atomic/Input/CHANGELOG.md

```
@@ -2,6 +2,13 @@
 
 The Input component allows the user to enter or edit information in text format.
 
+## 2025-08-20 `2.6.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)